### DISCUSSION TITLE
전면적인 Storybook 관련 세팅 수정 내용 반영을 위한 develop 브랜치로의 push 요청

### DISCUSSION THREAD

**Discussion by DragonTiger92** [FRONTENDSCHOOL8/Book-Kong#116] [Show and tell]
# 내용

 로컬 프로젝트 폴더에서 storybook 가동이 안 되어 확인해보니 package.json에 Storybook 관련 scripts가 설정이 안 되어 있어 Storybook 관련 script 코드를 추가하였습니다. 또한 Storybook 관련 세팅들이 추가로 필요하여 Storybook 관련 package들을 추가 설치하고 Storybook을 사용하기 위한 폴더를 프로젝트 루트 폴더에 새로 추가하였습니다.
 해당 수정 내용들을 commit하여 공용 repository의 develop 브랜치에 push하고 싶은데, 스크럼 마스터 님의 허락이 필요할 것 같아 discussion을 올립니다. 해당 내용 검토해주시면 감사하겠습니다.

## 변경 사항

- package.json에 Storybook 관련 코드 추가

  - Storybook 실행 script 외 Storybook 관련 script 추가

    - Storybook 실행 명령어: 'pnpm storybook'

  - Storybook addon들과 eslint-plugin-storybook package 추가

- .storybook 폴더와 내부 파일들 추가

- .eslintrc.cjs에 storybook 관련 세팅 추가

- Storybook 관련 package 추가 설치에 따른 pnpm-lock.yaml 파일 내용 변경

# 스크린샷

![image](https://github.com/FRONTENDSCHOOL8/Book-Kong/assets/148827389/1776cdc2-d78b-468d-be88-cb7876c46426)

**Comment by Samuel-0930** [comment]
넵!! 코드 코드 수정하신 부분은 따로 디스커션 없이 PR로 올려주심 될 것 같습니다!!

**Comment by DragonTiger92** [comment]
새로운 이슈 만들어서 이슈 내에서 develop 브랜치로부터 분기한 feature 브랜치 만들고 스토리북 세팅 관련 변경사항 PR 후 merge까지 완료
